src/doc/index.md

```diff
@@ -28,6 +28,7 @@ Rust provides a number of book-length sets of documentation, collectively
 nicknamed 'The Rust Bookshelf.'
 
 * [The Rust Programming Language][book] teaches you how to program in Rust.
+* [The Cargo Book][cargo-book] is a guide to Cargo, Rust's build tool and dependency manager.
```

Comment:
We're calling it "The Cargo Manual" in the cargo repo. Should we update that to say "The Cargo Book"?

Reply:
I'm okay with either, but the rest are almost all called "Book". I'll defer to @rust-lang/cargo here

Reply:
I'm fine with "The Cargo Book", it jives better with me with "bookshelf"

Reply:
Cool! Let's go with what you have here and I'll submit a PR for cargo to update the title there.

Reply:
PR to update cargo: https://github.com/rust-lang/cargo/pull/4695